test/Microsoft.DotNet.ProjectJsonMigration.Tests/Rules/GivenThatIWantToMigratePackageDependencies.cs

```diff
@@ -258,6 +258,28 @@ public void It_auto_add_desktop_references_during_migrate()
             items[0].Include.Should().Be("System");
         }
 
+        [Fact]
+        public void It_migrates_test_projects_to_have_test_sdk_and_xunit_packagedependencies()
```

Comment:
This should be two separate tests:

One to add the test sdk
One to convert the xunit dependency to the new packages.